python/cuml/preprocessing/TargetEncoder.py

```diff
@@ -114,7 +118,7 @@ def __init__(self, n_folds=4, smooth=0, seed=42,
         self.train = None
         self.output_type = output_type
 
-    def fit(self, x, y):
+    def fit(self, x, y, fold_ids=None):
```

Comment:
Can you add `fold_ids` to the docstring?

Reply:
yeah, thank you for comments.